### PR TITLE
User can login using Facebook

### PR DESCRIPTION
**Pull request by tochman**
PT: https://www.pivotaltracker.com/story/show/159425944

```
As a user
In order to have a more convenient login
I would like to be able to login with Facebook
```

#### What I learned:

* That despite how many times you do a task, you can always learn a new thing or two
* That we don't need to overcomplicate things in the controller action for OAuth.

### CODE REVIEW

File: features/step_definitions/assertion_steps.rb [hunk 1]
```diff
@@ -1,3 +1,7 @@
 When("I am at the index page") do
     visit root_path
+end
+
+Then("I should be on the Landing page") do
+    expect(current_path).to eq root_path
```
Comment (SophieSjoberg) [inline features/step_definitions/assertion_steps.rb]: What is the difference between landing and index page in this case..?

Comment (SophieSjoberg) [inline features/step_definitions/assertion_steps.rb:6]: What is the difference between landing and index page in this case..?

File: features/visitor_can_sign_up_using_facebook_credentials.feature [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    In order to have a more convenient login
+    I would like to be able to login with Facebook
+
+    PT: https://www.pivotaltracker.com/story/show/159425944
+
+
+    Scenario: Visitor clicks on "Login with Facebook" and gets authenticated
+        Given I am at the index page
+        And I click on "Log in with Facebook"
+        Then I should be on the Landing page
```
Comment (SophieSjoberg) [inline features/visitor_can_sign_up_using_facebook_credentials.feature:13]: Same question as before? :)

File: features/step_definitions/assertion_steps.rb (hunk 1 above)
Comment (tochman) [reply features/step_definitions/assertion_steps.rb:6]: The Landing page and the index is the same thing but feels more in scope referring to the fist page of the application. The first page the user is presented with.

**Review by SophieSjoberg — APPROVED** [review]
LGTM 💯

**Pull request merged**